apps/site/src/app/(main)/apply/sections/Form/AgeInformation.tsx

```diff
@@ -18,21 +18,21 @@ export default function AgeInformation() {
 		<div className="flex flex-col gap-5 w-11/12">
 			<div className="flex flex-col gap-5">
 				<p className="m-0 text-lg">
-					Because of limitations imposed by UCI, we are legally not
-					allowed to host minors (those under 18) for IrvineHacks
-					2024. By answering yes, you affirm that you are and will be
-					18 years or older by January 26, 2024.
+					Because of limitations imposed by UCI, we are legally not allowed to
+					host minors (those under 18) for IrvineHacks 2025. By answering yes,
+					you affirm that you are and will be 18 years or older by January 10,
+					2025.
 				</p>
 				<p className="text-[#FF2222] m-0 text-lg">
-					We will be checking ID. If you are a minor, you will be
-					turned away at the door.
+					We will be checking ID. If you are a minor, you will be turned away at
+					the door.
 				</p>
 			</div>
 
 			<SimpleRadio
 				name="is_18_older"
 				values={yesNoOptions}
-				title="Will you be 18 years or older by January 26th, 2024?"
+				title="Will you be 18 years or older by January 10th, 2025?"
```

Comment:
Issue: as long as attendees are 18+ by the start of IH - January 24th, 2025 - they can attend